packages/css/src/components/tabs/tabs.scss

```diff
@@ -54,8 +53,9 @@
   }
 
   &[aria-selected="true"] {
-    background-color: var(--ams-tabs-button-selected-background-color);
+    box-shadow: var(--ams-tabs-button-selected-box-shadow);
     color: var(--ams-tabs-button-selected-color);
+    font-weight: var(--ams-tabs-button-selected-font-weight);
```

Comment:
This causes a layout shift. [Here's a workaround for that](https://css-tricks.com/bold-on-hover-without-the-layout-shift/)